mmtc: init at 0.2.6

figsoda opened the pull request:
###### Motivation for this change
mmtc is a minimal mpd terminal client that aims to be simple yet highly configurable 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

figsoda committed:
mmtc: init at 0.2.6

figsoda commented:
Result of `nixpkgs-review pr 105702` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mmtc</li>
  </ul>
</details>

figsoda commented:
Ofborg didn't seem to like it. 

> This PR does not cleanly list package outputs after merging.

What does it mean? What can I do to fix it?

SuperSandro2000 commented on pkgs/applications/audio/mmtc/default.nix:
```diff
@@ -0,0 +1,38 @@
+      "Minimal mpd terminal client that aims to be simple yet highly configurable";
+    homepage = "https://github.com/figsoda/mmtc";
+    license = licenses.mpl20;
+    maintainers = [ ];
```
Missing maintainers and platforms.

figsoda committed:
maintainers: add figsoda

figsoda committed:
mmtc: add maintainers

figsoda committed:
mmtc: add meta.changelog and meta.platforms
I'm not sure what platforms exactly is it going to work on,
so it's set to platforms.all so I can know which ones work

figsoda commented:
Ofborg still says the same thing. Does it have something to do with `nixpkgs-mozilla`? Since it depends on the nightly version of rust.

SuperSandro2000 commented on pkgs/applications/audio/mmtc/default.nix:
```diff
@@ -0,0 +1,40 @@
+  rust = (let
+    mozilla = fetchFromGitHub {
+      owner = "mozilla";
+      repo = "nixpkgs-mozilla";
```
I don't think this is acceptable in nixpkgs.

figsoda replied:
> I don't think this is acceptable in nixpkgs.

I guess that's why, I will close this then

SuperSandro2000 reviewed (commented):
Please squash your mmtc related commits together.
